src/Security/Member.php

```diff
@@ -1352,7 +1352,7 @@ public static function mapInCMSGroups($groups = null)
             return ArrayList::create()->map();
         }
 
-        if (count($groups) == 0) {
+        if (is_countable($groups) && count($groups) == 0) {
```

Comment:
To be honest, I'd rather just ensure that this variable is countable via logic rather than shiv in these polyfills. This is a common pattern that is widely used in the framework, so likely to occur in other places too.

I think you should just change the default argument value to an array in the function declaration. It's technically a semver break though, so for now maybe you could just add `if (!$group) { $group = []; }`, maybe even make a PR to update the method signature against the master branch for SS5 as well?

Reply:
My feeling too. If it's not countable this should be an `InvalidArgumentException` shouldn't it?

Reply:
Not countable or null that is. :)

Reply:
This warning is thrown with the default value of `null`